source/lex.tex

```diff
@@ -518,13 +514,38 @@
 \indextext{comment!\tcode{//}}%
 The characters \tcode{//} start a comment, which terminates immediately before the
 next new-line character.
+Each comment is replaced by one \unicode{0020}{space} character;
+new-line characters are retained.
```

Comment:
There should be no retention of new-line characters within C-style comments. Additionally, retention of new-line characters following the termination of a `//` comment does not require mention. The termination point of a comment of either form is made clear above.
```suggestion
Each comment is replaced by one \unicode{0020}{space} character.
```